### ISSUE TITLE
Missing Textures - Rogues and Warriors RPG Series

### ISSUE THREAD

**Issue by CanecaKun** [Sinytra/Connector#1203]
### Mod Name

rogues-1.0.7+1.20.1 (Rogues and Warriors RPG Series)

### Mod Homepage

https://modrinth.com/mod/rogues-and-warriors

### Minecraft version

1.20.1

### Describe the bug

Missing textures on warriors basic skill (right click)

### Steps to reproduce

1. Download forge-1.20.1-47.2.0-installer
2. Download rogues-1.0.7+1.20.1 and its dependencies :
spell_engine-0.15.6+1.20.1
azurelibarmor-fabric-1.20.1-2.0.3
3. Download Connectors and forge-fabric api:
ConnectorExtras-1.11.1+1.20.1
Connector-1.0.0-beta.44+1.20.1
fabric-api-0.92.2+1.11.8+1.20.1

4. Run the mod, equip the warriors book and right click to activate the "weapon throw" ability that is missing textures as in the image provided

### Logs

https://gist.github.com/CanecaKun/f1d63429dcaf3181860d4834ce587ae4

### Additional context

![image](https://github.com/Sinytra/Connector/assets/62015531/9053d7bc-1ec4-4f63-91ac-4064018061ae)

**Comment by freshSquilliam** [comment]
year later and im having this issue. did you ever find a fix?

**Comment by CanecaKun** [comment]
Nope, I ended up uninstalling the mod, i had some similar issues with the archers passive effects that were corrected after an update, but i don't recall this issue being fixed, I guess the best you can do is make sure the mods are updated and try switching shaders and other stuff
